tests/testthat/test-add_risktable.R

```diff
@@ -16,6 +16,12 @@
 #' T3.2 The attribute components[['visR_plot']] contains the plot used as input.
 #' T3.3 The attribute components contains the risktables, identified through the risktable titles.
 #' T3.4 The output has class `ggsurvfit`.
+#' T4. T4. The function accepts a numeric table.height value
+#' T4.1 An error when the table.height is not numeric
+#' T4.2 An error when the table.height is negative
+#' T4.3 An error when the table.height is larger than 1
+#' T4.4 No error when the table.height is numeric
+#' T4.5 No error when the default table.height is changed
 
```

Comment:
One more test where you show that the parameter change has an effect. This can be checked using vdiffr or decomposing the image and seeing if the rowgutter is respected.